static/src/stylesheets/module/facia/_container--video.scss

```diff
@@ -426,3 +426,16 @@ $video-width-desktop: 700px;
     bottom: 2px;
     color: $neutral-3;
 }
+
+.fc-item__video-container {
+    .youtube-media-atom {
+        z-index: 1;
+    }
+
+    .youtube-media-atom__iframe:not(.youtube__video-ready) {
+        ~ .youtube-media-atom__overlay {
+            visibility: hidden;
+            opacity: 0;       
```

Comment:
Opacity isn't actually needed anymore following @duarte changes in https://github.com/guardian/frontend/pull/15526. Opacity on the overlay was transitioned from 0 to 1. Now this no longer happens. I'll update this now.